spk/minio/Makefile

```diff
@@ -12,7 +12,7 @@ MAINTAINER = fgma
 DESCRIPTION = The 100% Open Source, Enterprise-Grade, Amazon S3 Compatible Object Storage.
 RELOAD_UI = yes
 DISPLAY_NAME = MinIO
-CHANGELOG = "Update to 2019-10-12T01-39-57Z built with go 1.13.1"
+CHANGELOG = "Update to 2019-12-17T23-16-33Z built with go 1.13.1"
```

Comment:
Please add the kind of update (this one is a `Major Feature Release`) as the timestamp versioning is not very expressive.

And maybe its worth to add the changelog:
**Breaking Change** Lots of new features have been implemented, so rolling upgrade is not supported. Please upgrade all servers.

I would omit `built with go 1.13.1` (there is no change since spk rev 2)

Reply:
Added in https://github.com/SynoCommunity/spksrc/pull/3825/commits/9e5ba6077374db6c32f82607e4ac6bd70b1352e0